docs/conf.py

```diff
@@ -15,7 +15,7 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = "Fromager"
-copyright = "2024, Fromager Authors"
+copyright = "%Y, Fromager Authors"
```

Comment:
I am not sure if this will work. should we use following instead?
```
from datetime import datetime
copyright = f"{datetime.now().year}, Fromager Authors"
```

Reply:
I tested this and looks like Sphinx automatically processes the copyright string through time.strftime(), so "%Y" is actually the correct way to have a dynamic year